sdk/eventhub/mock-hub/samples/typescript/readme.md

```diff
@@ -1,60 +0,0 @@
-# Azure Mock Hub utility library samples
-
-This sample shows how to use the Azure Mock Hub utility.
-
-## Prerequisites
-
-The samples are compatible with all [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule).
-
-You'll also need to generate a pfx certificate chain that the mock service
-will use to enforce TLS.
-
-## Self-signed certificate instructions
```

Comment:
do we need to save these self-signed certificate instructions somewhere else?

Reply:
good point, addressed in https://github.com/Azure/azure-sdk-for-js/pull/30634/commits/ceed4dc2e188f59a00bf35f26bcbe4a9d341e5a0